src/dbghelp.rs

```diff
@@ -357,7 +303,7 @@ pub fn init() -> Result<Init, ()> {
         }
         debug_assert!(lock != 0);
         let lock = lock as HANDLE;
-        let r = WaitForSingleObjectEx(lock, INFINITE, FALSE);
+        let r = WaitForSingleObjectEx(lock, INFINITE, 0);
```

Comment:
I do still prefer being able to see when something is actually only supposed to have two values.
```suggestion
        let r = WaitForSingleObjectEx(lock, INFINITE, false as BOOL);
```

Reply:
Whoops! I tried to change `TRUE` -> `1` back, but I apparently missed the `FALSE` -> `0` changes. I'll fix this (though probably using the windows_sys `FALSE` constant, to be consistent with using `TRUE`)